### DISCUSSION TITLE
Forward agent for modular volttron

### DISCUSSION THREAD

**Discussion by bbartling** [VOLTTRON/volttron-core#154] [General]
When would the forward agent be ready for modular VOLTTRON? Then I think I can cut over my edge device officially : )

**Comment by craigpnnl** [comment]
@shwethanidd I looked at the milestones: https://github.com/orgs/eclipse-volttron/projects/8/views/2 and didn't find the forwarder agent.  Would that be something we do this quarter or after that?

**Reply by bbartling** [reply]
Hello any updates on this? @shwethanidd @craig8

**Reply by shwethanidd** [reply]
We'll start working on it soon. We need to first finish decoupling of auth and message bus completely. It should be ready by end of March.

**Reply by bbartling** [reply]
Thanks!
